charts/vault-operator/templates/apiregistration.yaml

```diff
@@ -1,8 +1,21 @@
+{{- $caCrt := "" }}
+{{- $serverCrt := "" }}
+{{- $serverKey := "" }}
+{{- if .Values.apiserver.servingCerts.generate }}
 {{- $ca := genCA "ca" 3650 }}
 {{- $cn := include "vault-operator.fullname" . -}}
 {{- $altName1 := printf "%s.%s" $cn .Release.Namespace }}
 {{- $altName2 := printf "%s.%s.svc" $cn .Release.Namespace }}
-{{- $cert := genSignedCert $cn nil (list $altName1 $altName2) 3650 $ca }}
+{{- $altName3 := printf "%s.%s.svc.cluster.local" $cn .Release.Namespace }}
```

Comment:
`.cluster.local` is configurable by the `--cluster-domain` flag to kubelet. https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/ . I think we should to hardcode this here.